packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -144,19 +152,6 @@
   @include mdc-text-field-dense-with-trailing-icon_;
 }
 
-.mdc-text-field--upgraded:not(.mdc-text-field--fullwidth):not(.mdc-text-field--box) {
```

Comment:
I'm confused as to why this was entirely removed? AFAICT the `--upgraded` modifier is no longer used if we remove this.

Historically we supported text field both CSS-only and with JS, which is why this modifier existed. Support for that was dropped more recently due to all of the work going into the filled and outlined variants. I suppose pursuing support for that again would need to be a separate effort anyway; if we've collapsed these styles into the defaults, we can remove all mention of the upgraded modifier in docs and JS.

I'm particularly wondering why the `height: 48px` disappeared though.

Reply:
I deleted all the upgraded CSS code since text-field wont be supporting CSS only.